spec/controllers/guests_controller_spec.rb

```diff
@@ -104,6 +105,33 @@
                                            firstname: "Test", lastname: "guest", email: "test@guest.com",
                                            occupation: "Other", department: "Other", gender: "Other", event_id: event.id }
           }.to change { Guest.count }.by(0)
+          expect(flash.alert).not_to be_nil
+        end
+      end
+
+      context "invalid request" do
+        it "re-renders new template" do
+          card_token = StripeMock.generate_card_token(last4: "4242", exp_year: 3001)
```

Comment:
3001 is a magic number that isn't explained. even though both of us know it's an arbitrary future date, it isn't explained in any capacity to others.